packages/dht/src/connection/ConnectionManager.ts

```diff
@@ -31,6 +32,7 @@ import { PendingConnection } from './PendingConnection'
 export interface ConnectionManagerOptions {
     maxConnections?: number
     metricsContext: MetricsContext
+    allowPrivateConnections: boolean
```

Comment:
Maybe this could be renamed as "allowIncomingPrivateConnections" to clarify that this is a server-side setting? This could also be given default value of false in order to avoid always passing this option when creating a ConnectionManager.